.github/workflows/unit-in-pull-request.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, windows, macos]
-        python-version: ["3.7"]
+        python-version: ["3.8"]
```

Comment:
Why?

Reply:
I got a little confused and thought that it would resolve the failing unit test - reverting